lib/model/game_state.dart

```diff
@@ -75,14 +75,14 @@ class GameState with ChangeNotifier {
   }
 
   void exileCurrentCard() {
-    exile.add(currentCard);
+    exile.add(currentCard!);
     currentCard = null;
 
     notifyListeners();
   }
 
   List<Card> getActiveQuestCards() {
-    final cardsToSearch = discard + deck + [currentCard];
+    final cardsToSearch = discard + deck + [currentCard!];
```

Comment:
Would be nice to revisit some of these `!` usages at some point, but for the purpose of this PR they're a pragmatic way to be able to move forwards. After all, it's no worse than the code was before, it's just making us aware of previously implicit assumptions :smile: